src/profiles/json.js

```diff
@@ -0,0 +1,44 @@
+
+function getJsonDocAsText() {
+  const membersDocId = getJsonDocId();
+  return DriveApp.getFileById(membersDocId).getAs('text/json').getDataAsString();
+}
+
+function getJsonDocId() {
+  //const members = convertSlidesFromPresentation(SlidesApp.openById(getTestPresentationId()));
+  const presentation = SlidesApp.getActivePresentation();
+  const documentProperties = PropertiesService.getDocumentProperties();
+  
+  var membersDocId = documentProperties.getProperty('MEMBERS_DOCUMENT_ID');  
```

Comment:
cannot find MEMBERS_DOCUMENT_ID property either on GUI or in the code. Now all the document properties are invisible to developers. Should:
* either give developers permission to access all the document properties 
* or store the properties names somewhere else